src/ensembl/src/content/app/entity-viewer/gene-view/components/unspliced-transcript/UnsplicedTranscript.tsx

```diff
@@ -54,11 +54,8 @@ It's possible that later we will switch to using UTRs for this purpose
 */
 
 const UnsplicedTranscript = (props: UnsplicedTranscriptProps) => {
-  const {
-    spliced_exons,
-    product_generating_contexts,
-    slice
-  } = props.transcript;
+  const { spliced_exons, product_generating_contexts, slice } =
+    props.transcript;
```

Comment:
Is this what Prettier is doing? I hate Prettier!

Reply:
Yes it is the culprit once more